dist/robotframework/lib/PhilipAPI.py

```diff
@@ -0,0 +1,21 @@
+from riot_pal import LLMemMapIf, PHILIP_MEM_MAP_PATH
+from robot.version import get_version
+from time import sleep
+
+class PhilipAPI(LLMemMapIf):
```

Comment:
Maybe we start without philip at all and just do non-philip tests to show as an example.